webui/react/src/utils/string.ts

```diff
@@ -47,6 +47,16 @@ export const generateLetters = (length = DEFAULT_ALPHA_NUMERIC_LENGTH): string =
   return generateAlphaNumeric(length, LETTERS);
 };
 
+export const generateUUID = (): string => {
```

Comment:
Since tests are running on Node / major browsers, let's `import crypto` to natively call `crypto.randomUUID()` and we won't need the test either

Reply:
ah nice, good call!

Reply:
doh! So it looks like crypto.randomUUID is not supported in node 14.16 which is what our CI is based off of. I don't really want to rock the boat by upgrading to 14.18 . I could do a polyfill but that one I function I had before would be a lot less to maintain then another library.

The other issue was that typescript doesn't have native definition for crypto.randomUUID until 4.6.2 (their most recent release) and when I upgrade to tag, there are a slew of other fixes that needed address (specifically the unknown error type handling, no longer forgiving)